zookeeper-client/zookeeper-client-c/CMakeLists.txt

```diff
@@ -189,6 +189,7 @@ endif()
 
 if(CYRUS_SASL_FOUND)
   list(APPEND zookeeper_sources src/zk_sasl.c)
+  set(LIBS ${LIBS} -lsasl2)
```

Comment:
This form creates a CMake "list," which ends up as a semi-separated string in the `libzookeeper.pc`.  So you could either continue using a list (in which case I'd suggest `list(APPEND LIBS ...)`), or format a string by quoting as in `set(LIBS "${LIBS} -lsasl2")`.  Yeah…

But in this case I'd suggest not adding those to `pkg-config`'s `Libs:` at all (as you'd then also have to figure out the proper `-L` flags, etc.) but rather as `Requires.private: libsasl2` (and, if OpenSSL is enabled, `Requires.private: [...] libssl`).